### PR TITLE
fix(SFINT-5094): Attempt to fix the quantic package release

### PR DESCRIPTION
**Pull request by erocheleau**

### CODE REVIEW

File: packages/quantic/scripts/build/util/sfdx.ts [hunk 1]
```diff
@@ -26,18 +26,20 @@ export function sfdx<T = SfdxResponse>(command: string): Promise<T> {
...
       (error, stdout) => {
-        if (error) {
-          let jsonOutput;
+        let jsonOutput: unknown;
+        if (stdout) {
           try {
             jsonOutput = JSON.parse(strip(stdout));
-          } catch (e) {
+            if (error) {
+              reject(jsonOutput || error);
```
Comment (nathanlb) [inline packages/quantic/scripts/build/util/sfdx.ts:34]: maybe we should throw here instead so that the stdout gets logged on line 39

Comment (nathanlb) [reply packages/quantic/scripts/build/util/sfdx.ts:34]: or just add logging here as well given we reject here with different info

**Review by louis-bompart — APPROVED** [review]

File: packages/quantic/scripts/build/util/sfdx.ts (hunk 1 above)
Comment (erocheleau) [reply packages/quantic/scripts/build/util/sfdx.ts:34]: added

File: packages/quantic/scripts/build/util/sfdx.ts [hunk 2]
```diff
@@ -26,18 +26,22 @@ export function sfdx<T = SfdxResponse>(command: string): Promise<T> {
...
+              console.error(jsonOutput);
+              console.error(error);
```
Comment (louis-bompart) [inline packages/quantic/scripts/build/util/sfdx.ts:36]:
```suggestion
              console.error(jsonOutput);
              console.error(error);
              reject(jsonOutput || error);
```

**Pull request merged**